great_expectations/core/usage_statistics/events.py

```diff
@@ -129,6 +129,7 @@ class UsageStatsEvents(enum.Enum):
     DATA_ASSISTANT_RESULT_GET_EXPECTATION_SUITE = (
         "data_assistant.result.get_expectation_suite"
     )
+    CLOUD_MIGRATE = "cloud_migrate"
```

Comment:
for consistency, should this be `class_name.method_name`?

Reply:
Good call - done!